### PR TITLE
Changing openapi refresh flag to false

### PR DESCRIPTION
**Pull request by timayabi2020**
﻿

Fixes #
This PR is supposed to fix issue [#1345](https://github.com/microsoftgraph/msgraph-sdk-powershell/issues/1345)

### Changes proposed in this pull request

- Updating the ForceRefresh flag to false since its causing error 500 during download of beta version of openapi. Weekly referesh pipeline fails at this stage
image
-The image below shows successful downloads with ForceRefresh set to false
image

### CODE REVIEW

File: tools/UpdateOpenApi.ps1 [hunk 1]
```diff
@@ -39,7 +39,7 @@ $ModuleMapping.Keys | ForEach-Object -Begin { $RequestCount = 0 } -End { Write-D
     $ForceRefresh = $false
     # Check whether ForceRefresh is required, Only required for the First Request.
     if ($RequestCount -eq 0) {
-        $ForceRefresh = $true
```
[the remark this answers is not in the document]
Comment (timayabi2020) [reply tools/UpdateOpenApi.ps1:42]: @peombwa yes it is. We need this to atleast have a refresh on the docs. I was hoping to use test (http://devxapitest.azurewebsites.net/) but it is also not stable. The staging one is also not stable.

[the remark this answers is not in the document]
Comment (peombwa) [reply tools/UpdateOpenApi.ps1:42]: Sounds good! So, we won't merge this PR once the build is complete? This is just to trigger generation of docs.

[the remark this answers is not in the document]
Comment (timayabi2020) [reply tools/UpdateOpenApi.ps1:42]: > Sounds good! So, we won't merge this PR once the build is complete? This is just to trigger generation of docs.

Yes. We won't merge

[the remark this answers is not in the document]
Comment (timayabi2020) [reply tools/UpdateOpenApi.ps1]: Cool. I'll go ahead and close it

**Pull request closed without merging**